.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Recipe
+about: This template makes sure relevant information is contained in a new recipe pull request.
+title: [Dataset Name]
+labels: ''
+assignees: ''
+
+---
+<!-- 
+👋 Thanks for opening a PR! This template describes the relevant information for a new recipe pull request.
+
+Appropriate files:
+Do you have a meta.yaml and a recipe.py in this PR? If you don't know what those are, please check out the sandbox https://mybinder.org/v2/gh/pangeo-forge/sandbox/binder or https://pangeo-forge.readthedocs.io/en/latest/introduction_tutorial/intro_tutorial_part3.html
```

Comment:
Let's indicate clearly that the meta.yaml and recipe.py should be in a subdirectory in the `recipes` directory, e.g. `recipes/my-new-feedstock`.

Reply:
good idea